twopane-navigation/src/components/paneRenderer/PaneRenderer.tsx

```diff
@@ -73,16 +73,38 @@ const PaneRenderer = (props: IPaneRendererProps) => {
         
     }
 
-// TODO MOVE TO ONE RENDER
+    const isComponentDisplayed = (component: IPaneComponent) => {
+        const isTopPaneOne =
+            component.pane === paneType.ONE &&
+            paneOneComponents.pop()?.key === component.key;
+        const isTopPaneTwo =
+            component.pane === paneType.TWO &&
+            paneTwoComponents.pop()?.key === component.key;
+        /*
+            If app is in single screen mode and component is at the top of the pane one stack, return true.
+            If app is in two screen mode and component is at the top of pane one stack, return true.
+                If component is at the top of pane two stack and pane one is not extended across both screens, return true.
+        */
+        if (
+            (paneRects.length === 1 && isTopPaneOne) ||
+            (paneRects.length > 1 &&
+                (isTopPaneOne || (isTopPaneTwo && !isPaneOneExtended())))
+        ) {
+            return true;
+        } else {
+            return false;
+        }
+    };
+
     return (
         <Fragment>
             {
                 paneComponents.map((val: IPaneComponent) =>
-                <View key={val.key}>
+                    <View importantForAccessibility={isComponentDisplayed(val) ? 'auto' : 'no-hide-descendants'} key={val.key}>
```

Comment:
nit: the extra space is still here. you can delete if by using backspace and then inserting a tab, I think; same on line 120